client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/test/boost-site-performance.test.tsx

```diff
@@ -43,41 +43,73 @@ describe( 'BoostSitePerformance', () => {
 		expect( strongTag ).toHaveStyle( 'font-weight: bold' );
 	} );
 
-	test( 'renders the Optimize css button when there is a score and has boost', () => {
-		render(
-			<BoostSitePerformance site={ site } trackEvent={ trackEventMock } hasError={ false } />
-		);
+	test( "renders the 'Auto-optimize' and 'Settings' buttons when there is a score and has no boost", () => {
+		const props = {
+			site: {
+				...site,
+				has_boost: false,
+				jetpack_boost_scores: { overall: 90, mobile: 90, desktop: 90 },
+			},
+			trackEvent: trackEventMock,
+			hasError: false,
+		};
 
-		const button = screen.getByRole( 'link', { name: /optimize css/i } );
-		expect( button ).toBeInTheDocument();
-		expect( button ).toHaveAttribute(
+		render( <BoostSitePerformance { ...props } /> );
+
+		const autoOptimizeButton = screen.getByText( /Auto-optimize/i );
+		expect( autoOptimizeButton ).toBeInTheDocument();
```

Comment:
I have skipped mocking a click event here as there are a couple of things we would import (react-redux & query) library to make the unit test work.